recipes/r-snftool/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = '2.2.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-snftool
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: SNFtool_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/SNFtool_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/SNFtool/SNFtool_{{ version }}.tar.gz
+  sha256: 94209d4891c00e0039a69cf132643080e47b0ad98528c887c5d4eebee7c6a845
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-heatmap.plus
+
+  run:
+    - r-base
+    - r-heatmap.plus
+
+test:
+  commands:
+    - $R -e "library('SNFtool')"  # [not win]
+    - "\"%R%\" -e \"library('SNFtool')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=SNFtool
+  license: GPL
+  summary: Similarity Network Fusion takes multiple views of a network and fuses them together
+    to construct an overall status matrix. The input to our algorithm can be feature
+    vectors, pairwise distances, or pairwise similarities. The learned status matrix
+    can then be used for retrieval, clustering, and classification.
+  license_family: GPL
```

Comment:
Can you add a license_file here?

Reply:
Fair enough. Maybe we can add an issue to the feedstock and point upstream to it?

Reply:
I'll try contacting them, maybe they can clarify.